FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Models/FhirSearch/TypeFilter.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Models.FhirSearch
+{
+    public class TypeFilter
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TypeFilter"/> class.
+        /// We will trigger a http request for each typeFilter,
+        /// </summary>
+        /// <param name="resourceType">resource type</param>
+        /// <param name="parameters">query parameters</param>
+        public TypeFilter(string resourceType, IList<Tuple<string, string>> parameters)
+        {
+            ResourceType = resourceType;
```

Comment:
Does the `resourceType` need to be *null-or-whitespace* checked here? Or it will be checked in another place?

Reply:
null-or-whitespace check is added here.